config/initializers/devise.rb

```diff
@@ -96,7 +96,7 @@
   # a value less than 10 in other environments. Note that, for bcrypt (the default
   # encryptor), the cost increases exponentially with the number of stretches (e.g.
   # a value of 20 is already extremely slow: approx. 60 seconds for 1 calculation).
-  config.stretches = Rails.env.test? ? 1 : 12
+  config.stretches = Rails.env.test? ? 1 : 100_000
```

Comment:
I went with 100_000 based on a comment from @alex but it does seem like `200_000` would get us closer to what we have with bcrypt now. ref https://github.com/18F/identity-private/issues/735#issuecomment-251279978